<ui version="4.0" >
 <class>mainForm</class>
 <widget class="QMainWindow" name="mainForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>qTVg</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <widget class="QTabWidget" name="tabWidget" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="manageTab" >
       <attribute name="title" >
        <string>TV番組データ管理</string>
       </attribute>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="2" column="0" >
         <widget class="QGroupBox" name="grpLog" >
          <property name="title" >
           <string>TV番組データ</string>
          </property>
          <property name="checkable" >
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="1" column="6" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item rowspan="2" row="0" column="5" >
            <widget class="QLabel" name="lblDate" >
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="6" >
            <widget class="QLabel" name="dataIndex_0" >
             <property name="text" >
              <string>(なし)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4" colspan="2" >
            <widget class="QCheckBox" name="checkBox_0" >
             <property name="text" >
              <string>地上波</string>
             </property>
            </widget>
           </item>
           <item row="3" column="6" >
            <widget class="QLabel" name="dataIndex_1" >
             <property name="text" >
              <string>(なし)</string>
             </property>
            </widget>
           </item>
           <item row="5" column="6" >
            <widget class="QPushButton" name="btnGet" >
             <property name="text" >
              <string>データ取得</string>
             </property>
            </widget>
           </item>
           <item row="4" column="6" >
            <widget class="QLabel" name="dataIndex_2" >
             <property name="text" >
              <string>(なし)</string>
             </property>
            </widget>
           </item>
           <item row="4" column="4" colspan="2" >
            <widget class="QCheckBox" name="checkBox_2" >
             <property name="text" >
              <string>BSデジタル</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4" colspan="2" >
            <widget class="QCheckBox" name="checkBox_1" >
             <property name="text" >
              <string>UHF/BS</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" >
            <widget class="QPushButton" name="btnToday" >
             <property name="text" >
              <string>今日</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="5" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="4" >
            <widget class="QPushButton" name="btnClear" >
             <property name="text" >
              <string>データ削除</string>
             </property>
            </widget>
           </item>
           <item rowspan="5" row="0" column="0" colspan="2" >
            <widget class="QCalendarWidget" name="calendarWidget" />
           </item>
           <item rowspan="6" row="0" column="2" colspan="2" >
            <widget class="Line" name="line" >
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="3" colspan="2" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QGroupBox" name="grpTvType" >
          <property name="title" >
           <string>地域</string>
          </property>
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QComboBox" name="cmbAreaId" />
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QGroupBox" name="grpDataDirectory" >
          <property name="title" >
           <string>データディレクトリ</string>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="1" column="1" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2" >
            <widget class="QPushButton" name="btnSelectDir" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>0</hsizetype>
               <vsizetype>0</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text" >
              <string>　ディレクトリ指定　</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>141</width>
               <height>16</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="3" >
            <widget class="QLineEdit" name="fldDataDirectory" />
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="consoleTab" >
       <attribute name="title" >
        <string>(業務用)</string>
       </attribute>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QTextBrowser" name="consoleView" />
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>444</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionShow_Version" />
    <addaction name="separator" />
    <addaction name="actionAbout_Qt" />
   </widget>
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionAbout_Qt" >
   <property name="text" >
    <string>About &amp;Qt...</string>
   </property>
  </action>
  <action name="actionShow_Version" >
   <property name="text" >
    <string>Show &amp;Version...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>fldDataDirectory</tabstop>
  <tabstop>btnSelectDir</tabstop>
  <tabstop>cmbAreaId</tabstop>
  <tabstop>calendarWidget</tabstop>
  <tabstop>btnToday</tabstop>
  <tabstop>checkBox_0</tabstop>
  <tabstop>checkBox_1</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>btnClear</tabstop>
  <tabstop>btnGet</tabstop>
  <tabstop>consoleView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnToday</sender>
   <signal>clicked()</signal>
   <receiver>calendarWidget</receiver>
   <slot>showToday()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>103</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel" >
     <x>116</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
